v22.2/cloud-storage-authentication.md

```diff
@@ -417,41 +488,125 @@ Service accounts in Google and Kubernetes refer to different resources. See [Goo
 
 To use assume role authentication, you will need at least two IAM roles:
 
-- An _identity service account_: the IAM service account you have associated with your Kubernetes service account.
+- An _identity service account_: the IAM service account associated with your Kubernetes service account.
 - An _operation service account_: the IAM service account to be assumed. This contains the permissions required to complete a CockroachDB operation.
 
-### Set up Google Cloud Storage workload identity
+For a step-by-step on how to set this up for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters, see the following steps.
+
+### Step 1. Set up the Google Cloud workload identity
+
+The first step to setting up workload identity authentication is different for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters. As you work through the step, ensure you use the relevant section for your cluster:
+
+- [{{ site.data.products.dedicated }}](#set-up-the-identity-service-account-for-dedicated-clusters)
+- [{{ site.data.products.core }}](#set-up-the-identity-service-account-for-self-hosted-clusters)
+
+#### Set up the identity service account for dedicated clusters
 
-Before completing the steps to run a bulk operation with assume role, it is necessary to create a identity service account for your Kubernetes service account to assume. Then, you must configure your CockroachDB pods to use the Kubernetes service account. You can complete all of these steps with Google's guide [Use Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+Each {{ site.data.products.dedicated }} cluster has a pre-configured IAM role for [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs) service accounts that act as the cluster's identity. In addition, the clusters have a unique, built-in functional IAM role, which you can configure as a trusted identity by giving it access to an operation service account. This allows you to have the built-in IAM role assume another role (or roles as part of a [chain](#google-cloud-storage-role-chaining)).
+
+The built-in IAM identity roles are as follows, which become part of your Google identity service account name:
+
+IAM role name | Operation
+--------------+------------
+`crl-dr-store-user` | Backup, Restore, Import, Export
+`crl-cdc-sink-user` | Changefeeds
+`crl-kms-user` | Encrypted backups and restores
```

Comment:
Dedicated clusters also have `crl-logexp-user` for byo logging infra. Is that in this conversation as well? cc @dhartunian

Reply:
also, cc @andf-crl who covers docs for that area.

Reply:
Question for eng: Do end users need to know `crl-logexp-user` the same way they need to know `crl-kms-user`? If it's just do that the `logexport` endpoint functions in dedicated, then I might suggest it does not belong here. It's not part of any of the [log export config steps](https://www.cockroachlabs.com/docs/cockroachcloud/export-logs.html). Just guessing && hope this helps.

Reply:
At the moment, log export has not been updated to use the workload identities but it will eventually. Once it does, customers will trust the workload identity instead of their cluster accounts when configuring their IAM roles.

Reply:
Ah, right, we'll need to give crdb to use crl-logexp-user (if at all we intend to use it this way), got it. Thanks for the clarification!